samples/hybrid-connections/java/simple-websocket-demo/README.md

```diff
@@ -0,0 +1,90 @@
+# 'Simple' Web Socket Hybrid Connection Sample 
+
+First, as described in the description [Here](https://github.com/bainian12345/azure-relay/tree/java-samples/samples/hybrid-connections/java), you will need to retrieve different parameters from the enviroment variable `RELAY_CONNECTION_STRING` for connection and authentication purposes:
+
+```	java
+	static final String CONNECTION_STRING_ENV_VARIABLE_NAME = "RELAY_CONNECTION_STRING";
+	static final Map<String, String> connectionParams = HybridConnectionUtil.parseConnectionString(System.getenv(CONNECTION_STRING_ENV_VARIABLE_NAME));
+	static final String RELAY_NAMESPACE = connectionParams.get("Endpoint");
+	static final String ENTITY_PATH = connectionParams.get("EntityPath");
+	static final String KEY_NAME = connectionParams.get("SharedAccessKeyName");
+	static final String KEY = connectionParams.get("SharedAccessKey");
+```
+
+For Hybrid Connection with authorization required, `TokenProvider` will be required to authenticate connections made to Azure Relay:
+
+```java
+TokenProvider tokenProvider = TokenProvider.createSharedAccessSignatureTokenProvider(KEY_NAME, KEY);
+```
+
+## Listener 
+
+Creates an instance of `HybridConnectionListener` which requires authentication from a `TokenProvider`: 
+
+```java
+HybridConnectionListener listener = new HybridConnectionListener(new URI(RELAY_NAMESPACE + ENTITY_PATH), tokenProvider);
+```
+
+Opens the connection to Azure Relay and listens for any incoming connections:
+
+```java
+listener.openAsync();
+```
+
+Once the listner is connected to the Hybrid Connection, it can start accepting web socket connections asynchronously, then returns the web socket instance once a connection is established:
+
+```java
+listener.acceptConnectionAsync();
```

Comment:
This code doesn't seem to indicate the need to capture the return value (though I think the CompletableFuture + no await equivalent contribute to making this harder).